pkgs/top-level/perl-packages.nix

```
@@ -1329,6 +1329,21 @@ let
     buildInputs = [ DBI ];
   };
 
+  AWSSignature4 = perlPackages.buildPerlModule {
+    pname = "AWS-Signature4";
+    version = "1.02";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/L/LD/LDS/AWS-Signature4-1.02.tar.gz";
+      sha256 =
+        "20bbc16cb3454fe5e8cf34fe61f1a91fe26c3f17e449ff665fcbbb92ab443ebd";
+    };
+    propagatedBuildInputs = with perlPackages; [ LWP TimeDate URI ];
```

```suggestion
    propagatedBuildInputs = [ LWP TimeDate URI ];
```

`perlPackages` is already available.

---

Updated!